### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by alex-semenyuk**
changelog: none

### CODE REVIEW

File: clippy_lints/src/functions/mod.rs [hunk 1]
```diff
@@ -180,7 +180,7 @@ declare_clippy_lint! {
     /// ### Known problems
     /// Of course, this lint assumes that `Result` is used
     /// for a fallible operation (which is after all the intended use). However
-    /// code may opt to (mis)use it as a basic two-variant-enum. In that case,
+    /// code may opt to (miss)use it as a basic two-variant-enum. In that case,
```
Comment (Serial-ATA) [inline clippy_lints/src/functions/mod.rs:183]:
```suggestion
    /// code may opt to (mis)use it as a basic two-variant-enum. In that case,
```

File: clippy_lints/src/serde_api.rs [hunk 2]
```diff
@@ -6,7 +6,7 @@ use rustc_session::{declare_lint_pass, declare_tool_lint};
 
 declare_clippy_lint! {
     /// ### What it does
-    /// Checks for mis-uses of the serde API.
+    /// Checks for miss-uses of the serde API.
```
Comment (Serial-ATA) [inline clippy_lints/src/serde_api.rs:9]:
```suggestion
    /// Checks for mis-uses of the serde API.
```

**Review by Serial-ATA — CHANGES_REQUESTED** [review]

File: clippy_lints/src/loops/manual_find.rs [hunk 3]
```diff
@@ -139,7 +139,7 @@ fn last_stmt_and_ret<'tcx>(
     if_chain! {
         // This should be the loop
         if let Some((node_hir, Node::Stmt(..))) = parent_iter.next();
-        // This should be the funciton body
+        // This should be the funcition body
```
Comment (dswij) [inline clippy_lints/src/loops/manual_find.rs:142]:
```suggestion
        // This should be the function body
```

**Review by dswij — CHANGES_REQUESTED** [review]
Thanks, just a small typo and it's good to merge

**Comment by alex-semenyuk** [comment]
Done

**Comment by dswij** [comment]
@bors r+ squash

Thanks!

**Comment by dswij** [comment]
The PR is merged in master, but it seems like squashing with bors is not closing the issue correctly at the moment.

I will be manually closing this one.

**Pull request closed without merging**